src/trusted/service_runtime/arch/x86_64/sel_ldr_x86_64.c

```diff
@@ -126,6 +126,21 @@ void  NaClPatchOneTrampolineCall(uintptr_t  call_target_addr,
   NaClApplyPatchToMemory(&patch_info);
 }
 
+void  NaClPatchRegTrampolineCall(uintptr_t  target_addr) {
+  struct NaClPatchInfo  patch_info;
+
+  NaClPatchInfoCtor(&patch_info);
+
+  patch_info.num_abs64 = 0;
+  patch_info.num_abs32 = 0;
+
+  patch_info.dst = target_addr;
+  patch_info.src = (uintptr_t) &NaClTrampolineRegRestore;
+  patch_info.nbytes = 128;
+
```

Comment:
Use of Constants: If the value 128 has a specific significance, consider defining it as a constant with a descriptive name.

Reply:
I think this is a good comment. Other places where we use magic numbers may be worth finding as well.